app/src/main/res/values/strings.xml

```diff
@@ -78,7 +78,7 @@
 
     <string name="superuser_toggle_notification">Notifications</string>
     <string name="superuser_toggle_revoke">Revoke</string>
-    <string name="superuser_policy_none">No app has asked for superuser permission yet.</string>
```

Comment:
The convention is to use plural here.

Reply:
But it's still correct.

Reply:
Insofar as it is possible to understand what it says, yes. It has meaning.
My problem is that it isn't localized to English.

Reply:
As a native English speaker it seems perfectly fine.

Reply:
I'll concede this would read better in the plural: "No apps have asked for superuser permissions yet."

Reply:
OK.

Reply:
I'll reopen it for you since clearly it's not abandoned after all. 😛

Reply:
Top stuff. I am even planning to improve the language in a later PR.